### PR TITLE
Fee schedules need company and client fee amounts

### LINKED ISSUE #639: Fee schedules need a company and client amount
`fee_schedule_ranges` require that there is a `company_fee_in_cents` and `client_fee_in_cents` 
The `fee_in_cents` will be calculated on the server side by summing the 2 new fields.

Order items will now also need to include `fee_schedule_range_id`, `company_fee_in_cents` and `client_fee_in_cents`

### PR DESCRIPTION
**Pull request by neonknight64**
+ Added company_fee_in_cents and client_fee_in_cents
+ The fee_in_cents is calculated from the company_fee_in_cents and client_fee_in_cents
+ Added a structure DisplayFeeScheduleRange so that the display of fee ranges only show the accumulated fee

Closes #639

### CODE REVIEW

File: db/migrations/00000000000021_order_items/up.sql [hunk 1]
```diff
@@ -6,6 +6,8 @@ CREATE TABLE order_items (
   event_id uuid null references events,
   quantity bigint not null,
   unit_price_in_cents bigint not null,
+  company_fee_in_cents bigint not null,
```
Comment (Krakaw) [inline db/migrations/00000000000021_order_items/up.sql]: These need to go into a new migration

File: db/migrations/00000000000024_fee_schedule_ranges/up.sql [hunk 2]
```diff
@@ -3,6 +3,8 @@ CREATE TABLE fee_schedule_ranges (
   fee_schedule_id uuid not null REFERENCES fee_schedules(id),
   min_price bigint NOT NULL,
   fee_in_cents bigint NOT NULL,
+  company_fee_in_cents bigint NOT NULL,
```
Comment (Krakaw) [inline db/migrations/00000000000024_fee_schedule_ranges/up.sql]: These also need to go into another new migration file

File: db/migrations/00000000000021_order_items/up.sql (hunk 1 above)
Comment (neonknight64) [reply db/migrations/00000000000021_order_items/up.sql]: Made a new migration: 00000000000045_add_company_fee_and_client_fee_to_order_items

File: db/migrations/00000000000024_fee_schedule_ranges/up.sql (hunk 2 above)
Comment (neonknight64) [reply db/migrations/00000000000024_fee_schedule_ranges/up.sql]: Made a new migration 00000000000046_add_company_fee_and_client_fee_to_fee_schedule_ranges

File: db/migrations/00000000000046_add_company_fee_and_client_fee_to_fee_schedule_ranges/up.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE fee_schedule_ranges
+  ADD company_fee_in_cents bigint NOT NULL,
```
[the remark this answers is not in the document]
Comment (neonknight64) [reply db/migrations/00000000000046_add_company_fee_and_client_fee_to_fee_schedule_ranges/up.sql]: Added default fees of 0

File: db/migrations/00000000000045_add_company_fee_and_client_fee_to_order_items/up.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE order_items
+  ADD company_fee_in_cents bigint NOT NULL,
```
[the remark this answers is not in the document]
Comment (neonknight64) [reply db/migrations/00000000000045_add_company_fee_and_client_fee_to_order_items/up.sql]: Added default fees of 0

File: db/migrations/00000000000045_add_company_fee_and_client_fee_to_order_items/down.sql [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (Krakaw) [inline db/migrations/00000000000045_add_company_fee_and_client_fee_to_order_items/down.sql]:
```suggestion
ALTER TABLE order_items
  DROP COLUMN company_fee_in_cents,
  DROP COLUMN  client_fee_in_cents;

File: db/migrations/00000000000046_add_company_fee_and_client_fee_to_fee_schedule_ranges/down.sql [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (Krakaw) [inline db/migrations/00000000000046_add_company_fee_and_client_fee_to_fee_schedule_ranges/down.sql]:
```suggestion
ALTER TABLE fee_schedule_ranges
  DROP COLUMN company_fee_in_cents,
  DROP COLUMN client_fee_in_cents;
```

**Pull request merged**